companies/zoom.md

```diff
@@ -0,0 +1,28 @@
+---
+fortune_rank:
+name: Zoom Video Communications, Inc.
+slug: zoom
+blm_statements:
+  - url:
```

Comment:
```suggestion
  - url: https://twitter.com/zoom_us/status/1266877451646283776
```

Reply:
The message did not specifically address BLM so I didn't place the link on line 6. In my research I saw many posts from people challenging Zoom (and JPMorgan Chase) for skirting the specific issue. Given that please tell me if the link should or should not be on line 6.

Reply:
Thank you for pointing this out! This makes me realize that our team needs to be a bit more clear about what we are collecting. We are looking for corporate statements in response to the murder of George Floyd, which may or may not explicitly say 'Black Lives Matter'. This one is from May 30 and mentions George Floyd so I would like to include it. :)

Reply:
Done.